src/components/pet/PetForm.jsx

```
@@ -0,0 +1,306 @@
+  const handleBreedSelect = (breedId, breedName) => {
+    setBreedId(breedId);
+    setBreedName(breedName);
+  };
+
+  const checkBirthDate = (birthDate) => {
+    const today = new Date();
+    const inputDate = new Date(birthDate);
+    const isValidDate =
+      /^\d{4}-\d{2}-\d{2}$/.test(birthDate) && !isNaN(inputDate);
+
+    if (inputDate > today || !isValidDate) {
```

Biome (1.9.4)

[error] isNaN is unsafe. It attempts a type coercion. Use Number.isNaN instead.

See the MDN documentation for more details.
Unsafe fix: Use Number.isNaN instead.


(lint/suspicious/noGlobalIsNan)